file_ingestions.Rmd

```diff
@@ -0,0 +1,93 @@
+```{r, include=FALSE}
```

Comment:
The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

The second line should be blank, followed by the author name(s).